crates/sui-core/src/checkpoints/metrics.rs

```diff
@@ -33,12 +37,36 @@ impl CheckpointMetrics {
                 registry
             )
             .unwrap(),
+            last_executed_checkpoint: register_int_gauge_with_registry!(
+                "last_executed_checkpoint",
+                "Last executed checkpoint",
+                registry
+            )
+            .unwrap(),
             checkpoint_errors: register_int_counter_with_registry!(
                 "checkpoint_errors",
                 "Checkpoints errors count",
                 registry
             )
             .unwrap(),
+            checkpoint_exec_errors: register_int_counter_with_registry!(
+                "checkpoint_exec_errors",
+                "Checkpoint execution errors count",
+                registry
+            )
+            .unwrap(),
+            checkpoint_exec_recv_channel_overflow: register_int_counter_with_registry!(
+                "checkpoint_exec_recv_channel_overflow",
+                "Count of the number of times the recv channel from StateSync to CheckpointExecutor has been overflowed",
+                registry
+            )
+            .unwrap(),
+            checkpoint_exec_lag: register_int_gauge_with_registry!(
+                "checkpoint_exec_lag",
+                "Checkpoint execution lag (highest_synced_checkpoint - highest_executed_checkpoint)",
+                registry
+            )
+            .unwrap(),
```

Comment:
As mentioned above we may not need this metrics since it can be derived from other metrics